libs/components/core/src/index.ts

```diff
@@ -119,6 +121,7 @@ export { SkyTrimModule } from './lib/modules/trim/trim.module';
 
 export { SkyUIConfigService } from './lib/modules/ui-config/ui-config.service';
 
+export { SkyViewkeeper } from './lib/modules/viewkeeper/viewkeeper';
```

Comment:
This is referenced via a deep import in the docs. Probably should be exported since it's referenced in other "internal" public APIs.